lib/aryn-sdk/aryn_sdk/partition/partition.py

```diff
@@ -101,6 +118,8 @@ def partition_file(
     if aryn_config is None:
         aryn_config = ArynConfig()
 
+    docparse_url = aps_url or docparse_url  # For backwards compatibility
```

Comment:
would prob change docparse_url to `Optional[str]=None` and write this as 
```python
if aps_url is not None:
    if docparse_url is not None:
        logging.warning("\"aps_url\" and \"docparse_url\" parameters were both set. \"aps_url\" is deprecated. Using \"docparse_url\".")
    else:
        logging.warning("\"aps_url\" parameter is deprecated. Use \"docparse_url\" instead")
        docparse_url = aps_url
if docparse_url is None:
    docparse_url = ARYN_DOCPARSE_URL
assert docparse_url is not None, "wtf"
```

Reply:
I've now implemented a variant of your suggested change.